ui/components/app/edit-gas-popover/edit-gas-popover.component.js

```diff
@@ -48,14 +49,15 @@ export default function EditGasPopover({
   const gasLoadingAnimationIsShowing = useSelector(
     getGasLoadingAnimationIsShowing,
   );
+  const isNetworkCongested = useSelector(getIsNetworkCongested);
 
   const showEducationButton =
     (mode === EDIT_GAS_MODES.MODIFY_IN_PLACE ||
       mode === EDIT_GAS_MODES.SWAPS) &&
     networkAndAccountSupport1559;
   const [showEducationContent, setShowEducationContent] = useState(false);
 
-  const [warning] = useState(null);
```

Comment:
I didn't see a need to use `useState` here but happy to do something else if there is a reason for it!

Reply:
Agree it is not useful, these variable `warning` even if not in state does not seems very useful.